.rubocop.yml

```diff
@@ -104,87 +106,31 @@ Naming/FileName:
 
 # We should try to use StandardError, but some errors like timeout inherit from Exception (beware)
 Style/RescueStandardError:
-  Exclude:
-    - "app/controllers/v0/id_card_attributes_controller.rb"
-    - "app/controllers/v0/sessions_controller.rb"
-    - "app/models/form_profile.rb"
-    - "app/models/mhv_account.rb"
-    - "app/services/mhv_accounts_service.rb"
-    - "app/uploaders/claim_documentation/uploader.rb"
-    - "app/workers/education_form/create_daily_spool_files.rb"
-    - "lib/common/client/concerns/monitoring.rb"
-    - "lib/common/client/configuration/soap.rb"
-    - "lib/common/exceptions/base_error.rb"
-    - "lib/github/github_service.rb"
-    - "lib/mvi/messages/find_profile_message_helpers.rb"
-    - "lib/mvi/responses/profile_parser.rb"
-    - "lib/saml/settings_service.rb"
-    - "lib/saml/ssoe_settings_service.rb"
-    - "rakelib/connectivity.rake"
-    - "rakelib/mvi.rake"
-    - "rakelib/redis.rake"
-    - "spec/request/authentication/standard_authentication.rb"
-
-# Variable naming style apparently doesn't like: something_2, something_3
-Naming/VariableNumber:
-  Exclude:
-    - "lib/hca/enrollment_system.rb"
-    - "spec/controllers/v0/post_911_gi_bill_statuses_controller_spec.rb"
-    - "spec/lib/preneeds/service_spec.rb"
+  EnforcedStyle: implicit
 
 # All kinds of issues with this cop right now, so disabling it.
 Style/FormatStringToken:
   Exclude:
     - "rakelib/mvi.rake"
 
-# This cop has an improvement for false positives in Mixins in future release, Exclude for now.
-# https://github.com/bbatsov/rubocop/issues/5448
-Rails/LexicallyScopedActionFilter:
-  Exclude:
-    - "app/controllers/concerns/filterable.rb"
-    - "app/controllers/v0/sessions_controller.rb"
 
 # Disabling this rule for now. We should consider changing.
 Lint/BooleanSymbol:
   Exclude:
-    - "app/swagger/schemas/gibct/institutions.rb"
-    - "spec/request/breakers_integration_spec.rb"
-    - "spec/request/statsd_middleware_spec.rb"
-    - "spec/support/authenticated_session_helper.rb"
+    - "app/swagger/schemas/gibct/institutions.rb" #Swagger blocks defines keys as :true, need to keep as is
 
-# Should consider enforcing this rule. Disabling for now.
+# Almost certainly you'll want to use Time since your app is probably dealing with current dates and times and it has support for timezones (system/local and utc), whereas DateTime just has offsets from UTC. However, if you need to deal with dates and times in a historical context you'll want to use DateTime to avoid making the same mistakes as UNESCO
```

Comment:
>same mistakes as UNESCO
out of curiostiy, what is this in reference to?

Reply:
Rubocop prefers Time or Date over using DateTime. Examples defined here in the rules: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/DateTime

Reply:
```suggestion
# Use `Time` when dealing with current dates and times; it has support for timezones (system/local and utc), whereas `DateTime` only supports offsets from UTC.
# However, if you need to deal with dates and times in a historical context you'll want to use DateTime to avoid making the same mistakes as UNESCO.
# See: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/DateTime
```